src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -485,6 +491,19 @@
       <LogicalName>System.Data.SqlClient.SqlMetaData.xml</LogicalName>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup  Condition="'$(TargetGroup)' == 'netcoreapp'">
+    <Compile Include="System.Data.SqlClient.TypeForwards.cs" />
+    <Reference Include="System.Runtime.Extensions" />
+    <Reference Include="System.Data.Common" />
+    <Reference Include="System.ComponentModel.Primitives" />
+    <Reference Include="System.Xml.ReaderWriter" />
+    <Reference Include="System.Runtime.InteropServices" />
+    <Reference Include="System.Transactions.Local" />
+    <Reference Include="System.Collections.NonGeneric" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' AND '$(TargetsWindows)' == 'true'">
```

Comment:
This seems redundant. You could change line 230 to add the netcoreapp conditional.